src/content/docs/mlops/integrations/openai-integration.mdx

```diff
@@ -75,24 +82,118 @@ openai.Completion.create(
     temperature=0 
 )
 ```
+</Step>
+</Steps>
+ </Collapser>
+
+
+<Collapser 
+    id="install-with-node" 
+    title="Install with Node.js"
+  >
+<Steps>
+<Step>
+### Install OpenAI [#install-openai-node]
+
+Use npm and run:
+
+```bash
+npm install @newrelic/openai-observability openai
+```
+</Step>
+<Step>
+### Getting started [#getting-started-node]
```

Comment:
@RyanKadri What else can I name this step? What exactly is happening here?

Reply:
I see in the python documentation section, you called the corresponding section "Modify your code". I think that sounds fine. Another wording could be something like "Instrument your code". But this section and that one are doing the same thing. Getting the customer to set up our instrumentation library in their codebase